lib/browser/api/web-contents.js

```diff
@@ -326,6 +326,17 @@ WebContents.prototype._init = function () {
     }
   })
 
+  this.on('-ipc-invoke', function (event, channel, args) {
+    event.reply = (result) => event.sendReply({ result })
+    event.throw = (error) => event.sendReply({ error: error.toString() })
+    this.emit('ipc-invoke', event, channel, ...args)
+    if (channel in ipcMain._invokeHandlers) {
```

Comment:
This will allow people to do _Bad Things_ ™️ with the object prototype.  As outlined above this should be changed to be a `Map` and a `has` check